pkgs/development/interpreters/erlang/generic-builder.nix

```diff
@@ -43,7 +44,7 @@
 , odbcSupport ? false
 , odbcPackages ? [ unixODBC ]
 , opensslPackage ? openssl
-, wxPackages ? [ libGL libGLU wxGTK xorg.libX11 ]
+, wxPackages ? [ libGL libGLU wxGTK xorg.libX11 wrapGAppsHook ]
```

Comment:
I think wrapGAppsHook should go in nativeBuildInputs, but this wxPackages goes in buildInputs.